1_fetch/src/get_nwis_data.R

```diff
@@ -21,14 +21,19 @@ download_nwis_data <- function(filepath, parameterCd = '00010',
   return(filepath)
 }
 
+
+## Combine the site csv's
+combine_sites <- function(site_csv, file_out){
+  site_csv %>% map_dfr(read_csv, col_types = 'ccTdcc') %>%
+  write_csv(.,file_out)
```

Comment:
Not a required change, just a comment! You can actually skip the `.,` part and leave it as `write_csv(file_out)` since the pipe automatically passes the previous data into the next function as the first argument.